internal/tiltfile/tiltfile.go

```diff
@@ -107,7 +112,20 @@ func makeSkylarkGitRepo(thread *skylark.Thread, fn *skylark.Builtin, args skylar
 		return nil, fmt.Errorf("Reading path %s: %v", path, err)
 	}
 
-	return gitRepo{absPath}, nil
+	ctx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
+	defer cancel()
+	t1, err := git.NewGitIgnoreTester(ctx, absPath)
```

Comment:
this will not error if the files do not exist, and instead return a matcher that matches nothing. (Like what happens if there's no `.gitignore` file in regular git.